simple_api/object/validators.py

```diff
@@ -0,0 +1,73 @@
+from inspect import isclass
+
+
+def build_validation_fn(action_validators, field_validators):
+    """
+    Prepares function for validation of input values and the action overall, instantiates validators which are not
+    already instantiated. Action_validators can be a sigle validator, Field_validators a List of tuples
+    (string, (validators...)) as prepared in object/actions.py
+    """
+    if action_validators and field_validators is None:
+        return None
+
+    if not isinstance(action_validators, (list, tuple)):
+        action_validators = action_validators,
```

Comment:
tohle by asi casem chtelo dat do utils, delani tuplu z jednoprvkove veci se deje nejak podezrele casto - zatim to asi muzes nechat byt, ale nejak se to mnozi

Reply:
Good idea, pak udělám